qiita_db/environment_manager.py

```diff
@@ -249,6 +249,9 @@ def make_environment(load_ontologies, download_reference, add_demo_user):
 def drop_environment(ask_for_confirmation):
     """Drops the database specified in the configuration
     """
+    # The transaction has an open connection to the database, so we need
+    # to close it in order to drop the environment
+    transaction.close()
```

Comment:
@ElDeveloper I don't think it is needed. The Transaction object re-opens the connection if it has been closed once you enter into the context (see [here](https://github.com/biocore/qiita/pull/1320/files#diff-29a37d36652014d33c798df45a644b46R738))